test/integration/015_cli_invocation_tests/test_cli_invocation.py

```diff
@@ -173,3 +234,55 @@ def _run_simple_dbt_commands(self, project_dir):
         # In case of 'dbt clean' also test that the clean-targets directories were deleted.
         for target in self.config.clean_targets:
             assert not os.path.isdir(target)
+
+
+class TestCLIInvocationWithProfilesAndProjectDir(ModelCopyingIntegrationTest):
+
+    @property
+    def schema(self):
+        return "test_cli_invocation_015"
+
+    @property
+    def models(self):
+        return "models"
+
+    @property
+    def custom_schema(self):
+        return "{}_custom".format(self.unique_schema())
+
+    def _test_postgres_sub_command_with_profiles_separate_from_project_dir(
+        self,
+        dbt_sub_command: str
+    ):
+        """
+        Test if a sub command runs well when a profiles dir is separate from a
+        project dir.
+        """
+        profiles_dir = "./tmp-profile"
+        workdir = os.getcwd()
+        with temporary_working_directory() as tmpdir:
```

Comment:
I made this a context manager as chaning the working directory could create weird side effects if it fails halfway (and therefor does not set the working dir back to the old one)